oas_docs/output/kibana.serverless.yaml

```diff
@@ -12086,7 +12116,7 @@ paths:
 
         The difference between the `id` and `rule_id` is that the `id` is a unique rule identifier that is randomly generated when a rule is created and cannot be set, whereas `rule_id` is a stable rule identifier that can be assigned during rule creation.
         > warn
-        > When used with [API key](https://www.elastic.co/docs/deploy-manage/api-keys) authentication, the user's key gets assigned to the affected rules. If the user's key gets deleted or the user becomes inactive, the rules will stop running.
+        > When used with [API key](https://www.elastic.co/docs/deploy-manage/api-keys) authentication, the user's key gets assigned to the affected rules and invalidates the old API key. If the user's key gets deleted or the user becomes inactive, the rules will stop running.
 
```

Comment:
Can we clarify what does it mean the rules will stop running? Will they error out? We don't disable them, right?